pkg/controller/storagecluster/noobaa_system_reconciler_test.go

```diff
@@ -395,3 +395,104 @@ func getReconciler(t *testing.T, objs ...runtime.Object) ReconcileStorageCluster
 		platform: &Platform{},
 	}
 }
+
+type testKMSArgs struct {
+	kmsProvider string
+	kmsAddress  string
+	testLabel   string
+	valid       bool
+}
```

Comment:
Don't define a separate struct, just do an inline struct in the test function.

Reply:
Done..